### PR TITLE
Addressed issues discussed in team meet - this is ready for final review

### LINKED ISSUE #157: Add intake info to search
As an admin user
I would like to be able to filter my search by intake status (e.g. like the opps filter)
And also by mini and full intake dates
So that I can quickly select candidates who have completed intake stages

### PR DESCRIPTION
**Pull request by samschlicht**
This seems to work fine: linked candidate notes to candidate in server, and then added a method to set a transient field 'intaked' by iterating over the notes. 

I'm not looking to merge just yet, just want to check if this seems like a sensible approach... @camerojo in particular might take a view, since you already had something in mind?

Screenshot 2023-09-11 at 7 35 36 pm

### CODE REVIEW

**Comment by samschlicht** [comment]
Also plan to add hover-over date if first part all good.

**Review by camerojo — COMMENTED** [review]
Hi @samschlicht - this works well for displaying intaked in results. But the problem is more complicated that that. I believe that they want to have intaked as a search field. That is not possible to do using the Transient field. Also, it gets more complicated for elastic searches.

**Comment by samschlicht** [comment]
@camerojo Just noting that in consultation Mariam and Zeynep also expressed that seeing intaked candidates at a glance on regular searches would be useful as a standalone feature, so I'm splitting this ticket — will create a new one for the search filter, which is probably more important but may unfortunately have to wait till next release..

**Review by camerojo — APPROVED** [review]
Approved

File: ui/admin-portal/src/app/model/candidate-field-info.ts [hunk 1]
```diff
@@ -39,11 +39,13 @@ export class CandidateFieldInfo {
...
    * only displayable to user admins.
+   * @param sortable Designates whether a field is intended to be sortable, meaning we can include non-sortable (e.g. @Transient) fields and disable the frontend sort-on-click behaviour that would otherwise cause an error (using *ngIf).
    */
   constructor(displayName: string, fieldPath: string,
               private tooltipSupplier: (value: any) => string,
               private fieldFormatter: (value: any) => string,
-              public fieldSelector: () => boolean) {
+              public fieldSelector: () => boolean,
+              public sortable: boolean) {
     this.displayName = displayName;
```
Comment (camerojo) [inline ui/admin-portal/src/app/model/candidate-field-info.ts:49]: Well done @samschlicht on making this sortable feature "data driven"

**Pull request merged**